packages/theme/src/components/ThemeProvider.tsx

```diff
@@ -89,16 +89,18 @@ const ThemeProvider = ({
   );
 
   React.useEffect(() => {
-    const hydrogen = document.querySelectorAll(themeSelector || "[data-h2]");
+    const hydrogen = document.querySelectorAll(
+      themeSelector || "html[data-h2]",
+    );
     let themeString: string | undefined = "";
-    // TO DO: Add mode back once dark mode is done
     if (mode && key) {
-      // themeString = `${key} ${mode}`;
+      themeString = `${key} ${mode}`;
+      // TO DO: Remove this line to re-enable dark mode
```

Comment:
I tried removing L99 but I didn't see any difference.  Are there any stories that look different with theme mode changing?

Reply:
This should be just for the main application. I do not believe this will affect anything in storybook.